### PR TITLE
txn: added schema to conditions

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

File: pkg/ovsdb/transact.go [hunk 1]
```diff
@@ -377,14 +378,20 @@ func NewCondition(condition []interface{}) (*Condition, error) {
 	if !ok {
 		return nil, errors.New(E_INTERNAL_ERROR)
 	}
+	schema := tableSchema.LookupColumn(column)
```
Comment (roytman) [inline pkg/ovsdb/transact.go:381]: `schema` name can be misleading, what about `columnSchema` or `cSchema` ...

**Comment by roytman** [comment]
From previous PR, please add `omitempt` to "Checksm string                 `json:"checksm"`"

File: pkg/ovsdb/transact.go [hunk 2]
```diff
@@ -366,9 +366,10 @@ type Condition struct {
 	Column   string
 	Function string
 	Value    interface{}
+	Schema   *libovsdb.ColumnSchema
```
Comment (roytman) [inline pkg/ovsdb/transact.go:369]: s/Schema/ColumnSchema/

File: pkg/ovsdb/transact.go (hunk 1 above)
Comment (hunchback) [reply pkg/ovsdb/transact.go:381]: ok

File: pkg/ovsdb/transact.go (hunk 2 above)
Comment (hunchback) [reply pkg/ovsdb/transact.go:369]: ok

**Pull request closed without merging**